solana-programs/anchor/audius-data/tests/playlist.ts

```diff
@@ -339,41 +339,41 @@ describe("playlists", function () {
         baseAuthorityAccount,
         userId,
         bumpSeed,
-        adminStorageAccount: adminStorageKeypair.publicKey,
+        adminAccount: adminAccountKeypair.publicKey,
         id: randomId(),
         playlistMetadata,
         userAuthorityKeypair: newUserKeypair,
-        playlistOwnerPDA: newUserAcctPDA,
```

Comment:
i think if we keep `newUser` in other vars we should keep `newUser` for `newUserAccount[Address]` just to be consistent - if possible